### ISSUE TITLE
Check multiple with various combinations of names 

### ISSUE THREAD

**Issue by TechnoNotice** [alanrenouf/vCheck-vSphere#723]
We have multiple syslog servers configured on the hosts with a site specific server configured as a second target.

udp://syslog.domin.com:514,tcp://syslog-SITE1.sec.domain.com:514 OR udp://syslog.domin.com:514,tcp://syslog-SITE2.sec.domain.com:514

Is it possible to get vCheck to check both possibilities?  If so what would the format be?

**Comment by Sneddo** [comment]
I don't believe this is possible. It looks like the plugin essentially finds all syslog servers for a host and concatenate to a comma seperated string, which is then tested for it to be equal to the setting in the plugin.

Off the top of my head, you could potentially change the -ne to a -notmatch which should mean you could use regex in the $SyslogServer setting.
